app/api/orders.py

```diff
@@ -40,6 +40,16 @@
 alipay_blueprint = Blueprint('alipay_blueprint', __name__, url_prefix='/v1/alipay')
 
 
+def check_billing_info(data):
+    if data.get('payment_mode') in ['stripe', 'paypal', 'alipay', 'omise'] and not data.get('is_billing_enabled'):
```

Comment:
wouldn't `data.get('amount') > 0` be a better criteria to check for paid orders? @shreyanshdwivedi

Reply:
I think u are right. I thought that amount can be zero even due to discount code and got confused so didn't use this 
However u're right here. Will update it asap

Reply:
@uds5501 I thought about your suggestion but I think we should not use amount here. Placed tickets also have amount > 0 but we only need billing Info for the tickets paid via our platform. What do u think about this?

Reply:
In that case, I think this should work just fine.